### ISSUE TITLE
Open written SummaryWriter file and add information afterwards?

### ISSUE THREAD

**Issue by che85** [lanpa/tensorboardX#513]
Hi there, 
I am not sure if you mentioned it somewhere, but I was thinking about adding information about hyperparameters to my summaries. Problem is, that I already have 20+ training runs and I would just like to reopen those files and add the information to them. 

I can write a script for that, but is it possible to open and add information to an existing summary?

Thank you,
Christian

**Comment by lanpa** [comment]
Yes, you can specify the `logdir` when constructing `SummaryWriter`, where `logdir` is the folder name of previous run.

**Comment by che85** [comment]
@lanpa Thanks for the quick reply. I will look into it.
